main.go

```diff
@@ -304,9 +307,28 @@ func split(s string) (head, tail string) {
 
 func main() {
 	defer task.Exit()
+	ohFlags := flag.NewFlagSet("oh", flag.ContinueOnError)
```

Comment:
No real need for a FlagSet as this isn't for a subcommand. It would also be better to PanicOnError and have something that catches that panic and prints a usage message. This would also mean moving the flag parsing to its own function so that flag parsing panics aren't mixed up with other panics.

Reply:
I'll change it to use the default flagset instance. OTOH, I think the intended use of `flag.ContinueOnError` is to handle the Parse error in the calling program. Using `Parse()` in a goroutine and using `recover()` to handle a possible `panic()` inside seems too complex for what it's needed, no? am I missing something?

Reply:
Seems the default `FlagSet` is set to `flags.ExitOnError`, so that won't allow to do anything with the `Parse() `errors. Should I change it anyway?

Reply:
Here is a test program to demonstrate what I mean:
```
package main

import (
        "flag"
        "fmt"
)

func options() {
        defer func() {
                r := recover()
                if r != nil {
                        fmt.Println("%v")
                        flag.Usage()
                }
        }()

        command := flag.String("c", "", "parse next argument instead of stdin")

        flag.Parse()

        println(*command)
}

func main() {
        options()
}
```

Reply:
I understand what you mean, what I do not understand is why the default behavior of the package `flag` is not right. `flag.Parse` will print its error and the usage if something happens inside, and also will exit the program. I understand the panic / recover if we need to do something besides printing the usage and exiting.

Reply:
My mistake. For some reason I thought ContinueOnError was the default behavior. Yes, no need for the panic/recover and probably no need to have the option parsing in its own function.

Reply:
I merged `options()` into main again.